src/Demo.tsx

```diff
@@ -78,21 +78,24 @@ export default class Demo extends React.Component<void, DemoState> {
 
   makeColorList(colors: {}): JSX.Element {
     return (
-      <ul className="clearfix" style={{margin: "0", backgroundColor: `${this.state.lights ? "#fff" : "#000"}`}}>
+      <ul
+        className="clearfix"
+        style={{margin: "0", backgroundColor: `${this.state.lights ? "#fff" : "#000"}`}}
+      >
         { Object.entries(colors).map(color => this.makeColorSwatch(color)) }
       </ul>
     );
   }
 
-  makeColorSwatch(info: string[]): JSX.Element {
+  makeColorSwatch(info: string[] | {}): JSX.Element {
```

Comment:
Not a big deal since this is the demo, but I think you can fix this by declaring some types for the color constant, like 
```
interface ColorCodeMap {
  [key: string]: string];
}
interface ColorNameMap {
  [key: string]: ColorCodeMap;
}
```

I don't know what it's doing now since as far as I can tell the input is always a non-empty object, not a string array.